exercises/practice/zebra-puzzle/zebra_puzzle.h

```diff
@@ -1,10 +1,11 @@
 #ifndef ZEBRA_PUZZLE_H
 #define ZEBRA_PUZZLE_H
 
-/// Determine the nationality of the resident who drinks water.
-const char *drinks_water(void);
+typedef struct {
+   const char *drinks_water;
+   const char *owns_zebra;
+} solution_t;
```

Comment:
Can you update the type elsewhere? Now this doesn't match the prototype below or the tests.

Reply:
Sorry

Reply:
No worries!